src/perl/dmtest/DMTest/Separated.pm

```diff
@@ -24,22 +25,32 @@ my $log = Log::Log4perl->get_logger(__PACKAGE__);
 sub suite {
   my ($package) = assertNumArgs(1, @_);
 
+  my @testNames = ();
+  my $machine = undef;
   my $options = $package->makeDummyTest();
```

Comment:
It doesn't look like this $machine variable gets used for anything so you should just remove it from the function.

Reply:
fixed